### DISCUSSION TITLE
Can't add shortened words to dictionary

### DISCUSSION THREAD

**Discussion by DrReddP** [SubtitleEdit/subtitleedit#5075] [General]
No matter how many times I hit "add to user dictionary", the spell check will continue to flag words like sayin', talkin', etc. Any word with an apostrophe in place of a terminal letter never, ever gets added to the dictionary in such a way that spell check will stop flagging them. I just use "skip all", but that goes away when I open the next subtitle file.

Also is there a reason I can't attach an image to this post?

**Comment by niksedk** [comment]
Thx for the info - should now work in latest beta: https://github.com/SubtitleEdit/subtitleedit/releases/download/3.6.1/SubtitleEditBeta.zip

But also note, that you can use this option:
![image](https://user-images.githubusercontent.com/1261331/120222734-b973c680-c240-11eb-84ec-e42f42eb8395.png)


>Also is there a reason I can't attach an image to this post?

Just copy/paste...

**Reply by DrReddP** [reply]
I've dowloaded the beta, but where do I find this settings menu in your photo?

![image](https://user-images.githubusercontent.com/82276985/120224052-a943f300-c210-11eb-9e3a-abed90a23069.png)

**Comment by niksedk** [comment]
Try Options - Settings - Tools

**Reply by DrReddP** [reply]
It's not working on this word.
![image](https://user-images.githubusercontent.com/82276985/125143319-518a8880-e0e8-11eb-8235-21ccde52b154.png)

**Comment by DrReddP** [comment]
Got it. Looks like this setting exists in the non-beta as well. Thank you
